hw/mcu/stm/stm32_common/src/hal_nvreg.c

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <hal/hal_nvreg.h>
+#include <mcu/stm32_hal.h>
+
+#if defined RTC_BACKUP_SUPPORT
```

Comment:
I think you can use this define to protect the whole file, because it's failing on the F1 BSPs on Travis.

Reply:
Otherwise you could as well use `#if !MYNEWT_VAL(MCU_STM32F1)` if the above suggestion doesn't work!

Reply:
Ah yes, I'm finally understanding.
The bluepill one has RTC_BKP_NUMBER defined since it has some other backup register thing.
I'll use RTC_BACKUP_SUPPORT as you suggested.